### ISSUE TITLE
Compile-time warnings

### ISSUE THREAD

**Issue by Unisay** [purescript-contrib/purescript-quickcheck-laws#31]
```
> pulp --version
Pulp version 12.0.1
purs version 0.11.6 using /usr/bin/purs
```

Compile time warnings:
```
Warning 1 of 5:

  at bower_components/purescript-quickcheck-laws/src/Test/QuickCheck/Laws.purs line 22, column 16 - line 22, column 54

    The derived newtype instance for
                             
      Data.Enum.BoundedEnum A
                             
    does not include a derived superclass instance for Data.Enum.Enum.


  See https://github.com/purescript/documentation/blob/master/errors/MissingNewtypeSuperclassInstance.md for more information,
  or to contribute content related to this warning.

Warning 2 of 5:

  at bower_components/purescript-quickcheck-laws/src/Test/QuickCheck/Laws.purs line 33, column 16 - line 33, column 54

    The derived newtype instance for
                             
      Data.Enum.BoundedEnum B
                             
    does not include a derived superclass instance for Data.Enum.Enum.


  See https://github.com/purescript/documentation/blob/master/errors/MissingNewtypeSuperclassInstance.md for more information,
  or to contribute content related to this warning.

Warning 3 of 5:

  at bower_components/purescript-quickcheck-laws/src/Test/QuickCheck/Laws.purs line 44, column 16 - line 44, column 54

    The derived newtype instance for
                             
      Data.Enum.BoundedEnum C
                             
    does not include a derived superclass instance for Data.Enum.Enum.


  See https://github.com/purescript/documentation/blob/master/errors/MissingNewtypeSuperclassInstance.md for more information,
  or to contribute content related to this warning.

Warning 4 of 5:

  at bower_components/purescript-quickcheck-laws/src/Test/QuickCheck/Laws.purs line 55, column 16 - line 55, column 54

    The derived newtype instance for
                             
      Data.Enum.BoundedEnum D
                             
    does not include a derived superclass instance for Data.Enum.Enum.


  See https://github.com/purescript/documentation/blob/master/errors/MissingNewtypeSuperclassInstance.md for more information,
  or to contribute content related to this warning.

Warning 5 of 5:

  at bower_components/purescript-quickcheck-laws/src/Test/QuickCheck/Laws.purs line 66, column 16 - line 66, column 54

    The derived newtype instance for
                             
      Data.Enum.BoundedEnum E
                             
    does not include a derived superclass instance for Data.Enum.Enum.


  See https://github.com/purescript/documentation/blob/master/errors/MissingNewtypeSuperclassInstance.md for more information,
  or to contribute content related to this warning.

```
